hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/validation/country/ResponseFromCountryValidator.java

```diff
@@ -5,15 +5,11 @@
 import uk.gov.ida.saml.core.errors.SamlTransformationErrorFactory;
 import uk.gov.ida.saml.core.validation.SamlTransformationErrorException;
 import uk.gov.ida.saml.core.validation.SamlValidationSpecificationFailure;
-import uk.gov.ida.saml.hub.transformers.inbound.SamlStatusToIdpIdaStatusMappingsFactory;
 import uk.gov.ida.saml.hub.validators.response.idp.components.EncryptedResponseFromIdpValidator;
 
 public class ResponseFromCountryValidator extends EncryptedResponseFromIdpValidator {
 
-    public ResponseFromCountryValidator(
-            final SamlStatusToIdpIdaStatusMappingsFactory statusMappingsFactory) {
-        super(statusMappingsFactory);
-    }
+    public ResponseFromCountryValidator() { }
```

Comment:
Java by default provides a no arg no nothing ctor, so this should not be needed.

Reply:
This class will be made to override the new unmarshaller base class in #151 so the empty constructor will disappear

Reply:
👍